tests/python/test_sparse_activate.py

```diff
@@ -0,0 +1,34 @@
+import taichi as ti
+
+
+@ti.archs_support_sparse
+def test_pointer():
+    ti.init(print_ir=True)
+
+    x = ti.var(ti.f32)
+    s = ti.var(ti.i32)
+
+    n = 16
+
+    ptr = ti.root.pointer(ti.i, n)
+    ptr.dense(ti.i, n).place(x)
```

Comment:
nit: Actually WDYT if we remove this `dense` layer, i.e. `ptr.place(x)`? If so, `s[None]` should be exactly how many elements in `x` that are activated. It makes the test more obvious?

Reply:
Maybe we want to test the behavior of multi-layer sparse, i.e. make OpenGL fake `dynamic` support fail to test.